_pkgdown.yml

```diff
@@ -1,47 +0,0 @@
-url: https://forestgeo.github.io/allodb/
```

Comment:
This seemed to be unused, outdated, and corrupted (formatting issues). The easiest solution for now was to remove it. An empty _pkgdown.yml still works -- it just produces a default site. Look at it by opening data/index.html on your web browser.

Reply:
Thank you so much @maurolepore  on all this. As you may have noticed we merged your pull request and tried to fix some functions for a general improve. I still have few questions about publishing the website:

1. Which Branch/folder should I select? If I choose master/root then the website looks as I built it using the corrupt _pkgdown.yaml file (live now). if I choose master/docs it looks like the default built as shown on docs/index.html. I read the "Learn more" on Github pages, it seems it is not recommended to built a website from docs, correct?.

![image](https://user-images.githubusercontent.com/26872628/92627488-6fede080-f299-11ea-99eb-fb7b6b57cbb0.png)


2. I would like the final website to look like docs/index/html but change "References" to Functions" and add "Datasets" to include descriptions of tables (currently in References). I understand I need to add more tabs in the navigation bar with navbar. For that, do I write in the new _pkgdown.yaml file in root?

Another big learning curb!

Reply:
For now I would choose /docs. As you say, that will expose the default website at the URL https://forestgeo.github.io/allodb/. In my experience, having the site in /docs is fine -- as long as you remember to update it with `pkgdown::build_site()` (see https://pkgdown.r-lib.org/reference/build_site.html). 

I prefer to rebuild the site automatically with every push; and do that with this GitHub Action: https://github.com/r-lib/actions/tree/master/examples#build-pkgdown-site. Eventually you might want to do it this way but the result doesn't change -- it just leaves you with one thing less to worry about.

Exactly, to tweak the website you need to tweak the file _pkgdown.yaml. You may see the previous veersion as an example, then add a piece and rebuild with `pkgdown::build_site()`; repeating until you are happy. You can see examples of these file in other packages I wrote, e.g. https://github.com/forestgeo/fgeo/blob/master/_pkgdown.yml.